packages/ui-library/src/components/forms/textarea/index.ts

```diff
@@ -63,10 +63,28 @@ export class BlrTextarea extends LitElement {
   }
 
   protected updateCounter() {
+    const scrollTop = this.textareaElement?.scrollTop;
+    const isFocused = this.textareaElement === document.activeElement;
+
     const length = this.textareaElement?.value?.length;
     if (length !== undefined) {
       this.count = length;
     }
+
+    const shouldUpdateTextarea = this.textareaElement && scrollTop !== undefined;
+
+    if (shouldUpdateTextarea) {
+      requestAnimationFrame(() => {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-expect-error
+        this.textareaElement.scrollTop = scrollTop;
```

Comment:
i quite dont understand why this is needed...

Reply:
"requestAnimationFrame" ensures that the update to the scroll position is synchronized with the browser's rendering cycle. This method waits for the browser to be ready to render the next frame before making changes to the DOM.

"this.textareaElement.scrollTop = scrollTop"; is to preserve the current scroll position of the <textarea>. The scrollTop variable captures the scroll position of the textarea before any update

In summary, this code effectively resolves the Firefox-specific issue of the scrollbar jumping to the top on user input